docs/common.py

```diff
@@ -62,7 +62,7 @@
 from target.versions import release,version,version_devel
 
 # Other versions and variables unlikely to change on every point release
-release_1_1 = '1.1.0-rc.7'
+release_1_2 = '1.2.7.2'
```

Comment:
i pushed another commit with a few fixes (this and some typos) - let's make sure it doesn't get lost in the rebase/squash/merge/etc